tests/unit_tests/lcls_tools/common/image/test_processing.py

```diff
@@ -0,0 +1,93 @@
+import unittest
+import numpy as np
+
+from lcls_tools.common.image.processing import ImageProcessor
+from lcls_tools.common.image.roi import RectangularROI
+
+
+class TestImageProcessing(unittest.TestCase):
+    data_location: str = "tests/datasets/images/numpy/"
+
+    def __init__(self, methodName: str = "runTest") -> None:
+        super().__init__(methodName)
+        self.center = [400, 400]
+        self.size = (800, 800)
+        self.widths = (350, 300)
+        self.radius = 50
+        self.image = np.load(self.data_location + "test_roi_image.npy")
+
+    def test_process(self):
+        """
+        Given an np.ndarray and roi process
+        and assert the return in an np.ndarray
+        """
+        image_processor = ImageProcessor()
+        image = image_processor.process(self.image)
+        self.assertIsInstance(
+            image, np.ndarray,
+            msg="expected image to be an instance of np.ndarray"
+        )
+        roi = RectangularROI(
+            center=self.center, xwidth=self.widths[0], ywidth=self.widths[1]
+        )
+        image_processor = ImageProcessor(roi=roi)
+        image = image_processor.process(self.image)
+        self.assertIsInstance(
+            image, np.ndarray,
+            msg="expected image to be an instance of np.ndarray"
+        )
+        imageShape = image.shape
+        roiShape = (roi.ywidth, roi.xwidth)
+        self.assertEqual(
+            imageShape, roiShape,
+            msg=(f"expected image shape {imageShape} "
+                 + f"to equal roi {roiShape}")
+        )
+        # TODO:run coverage
+
```

Comment:
This is done right^?

Reply:
Yes! I left it in a comment above. I will remove this line.